src/power_grid_model_io/functions/phase_to_phase.py

```diff
@@ -40,30 +40,34 @@ def power_wind_speed(  # pylint: disable=too-many-arguments
     nominal_wind_speed: float = 14.0,
     cutting_out_wind_speed: float = 25.0,
     cut_out_wind_speed: float = 30.0,
+    axis_height: float = 30.0,
```

Comment:
Can we set the default to 10.0? As that results in the `(axis_height / 10) ** 0.143` being 1.0.